msm/src/mvlookup.rs

```diff
@@ -318,6 +508,11 @@ pub mod prover {
                 .iter()
                 .for_each(|evals| assert_eq!(evals.len(), domain.d1.size as usize));
 
+            // Sanity check to verify that we have all the evaluations for the fixed lookup tables
+            fixed_lookup_tables
+                .values()
+                .for_each(|evals| assert_eq!(evals.len(), domain.d1.size as usize));
```

Comment:
I suppose it is a repeated dummy value in the rows at the end to pad up to domain size if they are shorter?